docs/initrd.md

```diff
@@ -0,0 +1,44 @@
+# Creating and using an initrd for Firecracker
+
+## Creating
+
+### Based on alpine or suse
+
+You can use this script to generate an initrd either based on alpine or suse linux: https://github.com/marcov/firecracker-initrd
+
+The script extracts
```

Comment:
This sentence seems incomplete.

Reply:
Just finished the sentence. I think that's what I intended to write :)